ui/component/app/view.jsx

```diff
@@ -368,6 +372,17 @@ function App(props: Props) {
 
   useDegradedPerformance(setLbryTvApiStatus, user);
 
+  // Overlay page
+  if (pathname.includes('/$/overlay/')) {
+    /* This is needed otherwise the Overlay page will not work correctly. */
+    /* The Overlay page does not need every script since it will be used in OBS or others. */
+    return (
+      <React.Fragment>
+          <Router />
+      </React.Fragment>
+    );
+  }
+
```

Comment:
We probably shouldn't pollute the app code with these kind of changes if the overlay page (`page/overlay/*`) itself has nothing to do with the app and is not React code.

I think we can route this the same way `/$/rss` is being done (does not touch the app).

Reply:
The overlay is part of Odysee itself since it has a Overlay settings page which allows the User to update it to be used in their streams.
If you still feel the need to have it outside of the App, feel free to direct me to the code that allows it to be done.

This is the reason why the code in App view.jsx is there.

Without App view.jsx code.
![image](https://user-images.githubusercontent.com/19220996/148727346-1f5995a3-2934-45ae-a6ad-1dbccb519c3a.png)

With App view.jsx code.
![image](https://user-images.githubusercontent.com/19220996/148727437-6ab9c6cd-9f24-4692-a757-c56251e792ed.png)

Reply:
We don't need to go through the app and then make these kinds of hacked bypasses.
Take a look at `web/src/rss.js` -- you can move you code there (outside of the app).  
Then, add a route in `web/src/routes.js` to point to your overlay code.